lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -741,6 +800,19 @@ class HybridTextEditing {
       ..transform = transformCss;
   }
 
+  // TODO(flutter_web): When the browser closes and re-opens the virtual
+  // shifts the page in IOS. Call this method from visibility change listener
+  // attached to body.
+  /// Set the dom elements location somewhere outside of the screen.
+  ///
+  /// This is useful for not triggering a scroll when IOS virtual keyboard is
+  /// coming up.
+  ///
+  /// See [TextEditingElement._delayBeforePositining].
+  void setStyleOutsideOfScreen(html.HtmlElement domElement) {
```

Comment:
Is there a reason for this method to be in `HybridTextEditing`?

Reply:
Agreed, I'll also carry this method during the refactor.